app/assets/stylesheets/components/_background.scss

```diff
@@ -3,6 +3,11 @@
 .bg-lighter-blue { background-color: $blue-lighter; }
 .bg-lightest-blue { background-color: $blue-lightest; }
 .bg-lightest-red { background-color: $red-lightest; }
+// scss-lint:disable ImportantRule
+.sm-display-inline-block { display: inline-block !important; }
+// scss-lint:enable ImportantRule
+
```

Comment:
Is this used?

```suggestion
```

Reply:
omg.  lol.  grabbed that from another section but forgot to put bg white in